src/main/kotlin/com/terraformation/backend/tracking/db/PlantingSiteStore.kt

```diff
@@ -476,6 +476,15 @@ class PlantingSiteStore(
     )
   }
 
+  fun hasPlantings(plantingSiteId: PlantingSiteId): Boolean {
```

Comment:
There is a `hasSubzonePlantings` as well, not sure if it would be better to merge the two and expose additional arguments.